tensorboard/plugins/mesh/README.md

```diff
@@ -0,0 +1,127 @@
+static mesh:
+
+```python
+from tensorboard.plugins.mesh import summary as mesh_summary
+...
+
+point_cloud = tf.constant([[[0.19, 0.78, 0.02], ...]], shape=[1, 1064, 3])
+point_colors = tf.constant([[[128, 104, 227], ...]], shape=[1, 1064, 3])
+
+summary = mesh_summary.op('point_cloud', vertices=point_cloud, colors=point_colors)
+```
+
+**NOTE:** The `colors` tensor is optional in this case but can be useful to 
+show different semantics of the points.
+
+A mesh can be represented by a point cloud together with a set of faces,
+where each face is represented by a list of indices of vertices in
+clockwise order around the face. This plugin currently only supports
+triangular faces. Construct a mesh by including the `faces` parameter to
+`mesh_summary.op`:
+
+```python
+mesh = tf.constant([[[0.19, 0.78, 0.02], ...]], shape=[1, 1064, 3])
+colors = tf.constant([[[128, 104, 227], ...]], shape=[1, 1064, 3])
+faces = tf.constant([[[13, 78, 54], ...]], shape=[1, 752, 3])
+
+summary = mesh_summary.op('mesh', vertices=mesh, colors=colors, faces=faces)
+```
+
+The `colors` tensor is optional for mesh summaries.
+
+## Scene configuration
+
+Another aspect of displaying 3D visualizations is a scene configuration, i.e.,
+intensity and color of light sources, objects' materials, camera models, and so
+on. All of that can be configured via an additional parameter `config_dict`.
+This dictionary may contain three high-level keys: `camera`, `lights` and
+`material`. Each key must also be a dictionary with mandatory key `cls`,
+representing a valid [THREE.js](https://threejs.org) class name.
+
+```python
+camera_config = {'cls': 'PerspectiveCamera'}
+summary = mesh_summary.op(
+    "mesh",
+    vertices=mesh,
+    colors=colors,
+    faces=faces,
+    config_dict={"camera": camera_config},
+)
+```
+
+Consult the [THREE.js documentation](https://threejs.org/docs/index.html#manual/en/introduction/Creating-a-scene) for more details.
+All keys from `camera_config` will be passed to a class with name
+`camera_config.cls`. For example (based on the
+[`PerspectiveCamera` documentation](https://threejs.org/docs/index.html#api/en/cameras/PerspectiveCamera)):
+
+```python
+camera_config = {
+  'cls': 'PerspectiveCamera',
+  'fov': 75,
+  'aspect': 0.9,
+}
+...
+```
+
+Keep in mind that scene configuration is not a trainable variable (i.e., it is 
+a static attribute) and is fixed at summary-creation time.
+
+## How to install
+
+The mesh plugin isn’t yet part of stable TensorBoard, so you’ll need to 
+install the latest TensorBoard nightly build to use it.
+
+### Colab
+
+```
+!pip install -q -U tb-nightly
+```
+
+Then load TensorBoard extension and run it, similar to how you would do it in the Terminal:
+
+```
+%load_ext tensorboard
+%tensorboard --logdir=/path/to/logs
+```
+
+Please consult the [example Colab notebook](https://colab.sandbox.google.com/drive/1E6j9qXMGhxYb17diCR5qFMMk2YgC_Kk6) for more details.
```

Comment:
Please link to

https://colab.research.google.com/github/tensorflow/tensorboard/blob/master/tensorboard/plugins/mesh/Mesh_Plugin_Tensorboard.ipynb

instead of this fixed file owned by your user account. This way, we can
update the tutorial simply by updating the repository, and we don’t have
to worry about permissions or accidental editing.

You can check that this link works before the PR is merged by changing
`tensorflow` to `podlipensky` and `master` to `tf-graphics-demo`.